### PR TITLE
feat(terms): added at least 5 aliases per term + 3 new terms

### PR DESCRIPTION
**Pull request by iFlex**
Adde at least 5 aliases per term using variations of the words of interest for each term.

Added 3 more terms

### Requirements for making a pull request

Thank you for contributing to our project! 

Please fill out the template below to help the project maintainers review it as fast as possible and include your contribution to the project.

### What does it fix?

Updates https://github.com/code4romania/Alt-TaskBucket/issues/4#issue-581622547

Added at least 5 aliases for each of the terms in the JSON and added 3 new tersm.
Planing to add more terms but I'm not 100% sure this is exactly what you want, so doing this pull request now to see if I'm going in the right direction.

### How has it been tested?

Json was validated for correctness (ensured the data in the terms.json parses successfully). The project seems to have no test harness so no unit tests were run.

### CODE REVIEW

**Comment by catileptic** [comment]
Please remove Date la Zi and Diaspora Hub from the links since these websites have not been launched as of the time of this PR.

Please remove keywords that do not contain some variation of "covid-19" until #35 is implemented. 

I've submitted the terms to our content team for review. I expect they will answer by tomorrow morning (maximum waiting time0.

File: src/terms.json [hunk 1]
```diff
@@ -19,32 +19,53 @@
...
+        "title": "Disapora",
+        "aliases": ["disapora","ce mă fac diaspora","ce mă fac în diaspora","sunt in deplasare","străinătate coronavirus"],
+        "explanation": "<p>Dacă sunteți în străinătate și doriți să aflați informații legate de prevenirea infectării cu virusul COVID-19 (Coronavirus), puteți apela numărul de telefon:+4021.320.20.20</p><p>Mai multe informații găsiți aici:</p>",
+        "links": ["https://diasporahub.ro/","http://www.mae.ro/node/51961"]
+    },
+    "statistici":{
+        "title": "Statistici Coronavirus",
+        "aliases": ["statistici coronavirus","statistici covid-19","statistici cazuri coronavirus","statistică coronavirus","statistici cazuri covid-19"],
+        "explanation": "<p>Dacă doriți să consultați date statistice legate de coronavirus găsiți datele la zi aici:</p>",
+        "links": ["https://datelazi.ro/"]
```
Comment (tudoramariei) [inline src/terms.json]: Until we launch `datelazi`, I think it's OK to link to INTSP's dashboard 
https://instnsp.maps.arcgis.com/apps/opsdashboard/index.html#/5eced796595b4ee585bcdba03e30c127

@catileptic can approve this idea

File: src/terms.json [hunk 2]
```diff
@@ -19,32 +19,53 @@
...
-            "https://www.presidency.ro/ro/media/declaratii-de-presa/declaratia-de-presa-sustinuta-de-presedintele-romaniei-domnul-klaus-iohannis1584201636"
+            "https://www.presidency.ro/ro/media/declaratii-de-presa/declaratia-de-presa-sustinuta-de-presedintele-romaniei-domnul-klaus-iohannis1584201636",
+            "https://www.mai.gov.ro/wp-content/uploads/2020/03/Ordonan%C8%9Ba-militar%C4%83-nr.-1-2020-m%C4%83suri-de-prima-urgen%C8%9B%C4%83-Decret.pdf"
         ]
+    },
+    "cazuri":{
+        "title": "Cazuri de Coronavirus",
+        "aliases": ["cazuri","cazuri coronavirus","cazuri de coronavirus","număr infectări coronavirus","număr infecții coronavirus","cazuri covid-19","cazuri de covid-19","număr infectări covid-19","număr infecții covid-19"],
+        "explanation": "<p>Dacă doriți să consultați situația cazulior de cronavirus in timp real, informații găsiți aici:</p>",
+        "links": ["https://datelazi.ro/"]
```
Comment (tudoramariei) [inline src/terms.json]: I think it's the same case as with the comment below

File: src/terms.json [hunk 3]
```diff
@@ -19,32 +19,53 @@
...
+    },
+    "diaspora":{
+        "title": "Disapora",
+        "aliases": ["disapora","ce mă fac diaspora","ce mă fac în diaspora","sunt in deplasare","străinătate coronavirus"],
+        "explanation": "<p>Dacă sunteți în străinătate și doriți să aflați informații legate de prevenirea infectării cu virusul COVID-19 (Coronavirus), puteți apela numărul de telefon:+4021.320.20.20</p><p>Mai multe informații găsiți aici:</p>",
+        "links": ["https://diasporahub.ro/","http://www.mae.ro/node/51961"]
```
Comment (tudoramariei) [inline src/terms.json]: As per [catileptic's comment](https://github.com/code4romania/emergency-news-addon/pull/34#issuecomment-600700952) also remove `diasporahub` for now, please

**Comment by catileptic** [comment]
The list of terms and the text that is displayed when they are detected is a collaborative effort between Code for Romania and our partners (DSU, ADR). As such, we are not taking PRs on this.

**Pull request closed without merging**